### PR TITLE
Test Login

### PR DESCRIPTION
**Pull request by aquena-code**

### CODE REVIEW

**Review by daylerVMT — APPROVED** [review]
its okey no conflicts

File: test/test_login.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# @test_retrieve_post.py Copyright (c) 2022 Jalasoft
```
Comment (ajurado91) [inline test/test_login.py:1]: File name in copyright is wrong

**Comment by AlbertLlanos** [comment]
the copyright not have the name feature

File: test/test_login.py (hunk 1 above)
Comment (AlbertLlanos) [reply test/test_login.py]: The error is solved

**Review by daylerVMT — APPROVED** [review]
okey correct copy

**Pull request merged**